src/pages/frequently-asked-questions.tsx

```diff
@@ -0,0 +1,524 @@
+import Section from "@/components/basicInformation/Section";
+import Image from "next/image";
+import { useState } from "react";
+import { ReactNode } from "react";
+import NavbarItem from "@/components/basicInformation/utility/NavbarItem";
+import { text } from "stream/consumers";
+type Section = {
```

Comment:
Another minor TS nitpick, but it probably makes more sense to define the type in the same location as the original component, either in the component file itself or a separate `types.ts` file in the same directory. That way everything is kept in one place and it makes it easier to reuse the type in other files in future (such as in the `src/pages/what-is-domestic-violence.tsx` file). (Same with NavbarItem below).